src/util/config.js

```diff
@@ -6,3 +6,7 @@ export const API_BASE_URL =
 export const axiosInstance = axios.create({
   baseURL: API_BASE_URL,
 });
+
+// REACT_APP_FRONTEND_PROXY is using cart page for temporarily untill order page is created
```

Comment:
Maybe use the `window` object instead of creating another env variable? I don't think it should be a problem with Netlify.